cmd/run.go

```diff
@@ -303,6 +307,16 @@ func NewRunCommand(factory *factory.ProviderFactory, scenarioOrchestrator *scena
 			}
 
 			if !runDetached {
+				// capture stdout/stderr to both terminal and a log file for parsing
+				filename := fmt.Sprintf("%s.log", containerName)
+				file, cerr := os.Create(path.Clean(filename))
+				if cerr != nil {
+					spinner.Stop()
+					return cerr
+				}
+				defer file.Close()
+				mw := io.MultiWriter(os.Stdout, file)
```

Comment:
@abhinavs1920 please add a comment here to explain how the thing works because might be confusing. Explain that you're multiplexing a file and stdout to with a multiwriter  and that the container stdout and stderr will written in the machine stdout and in the file. Thanks!